crates/runtime/src/trampolines/x86_64.rs

```diff
@@ -1,115 +1,22 @@
-use wasmtime_asm_macros::asm_func;
-
 // Helper macros for getting the first and second arguments according to the
 // system calling convention, as well as some callee-saved scratch registers we
 // can safely use in the trampolines.
 cfg_if::cfg_if! {
     if #[cfg(windows)] {
-        macro_rules! arg0 { () => ("rcx") }
-        macro_rules! arg1 { () => ("rdx") }
+        macro_rules! callee_vmctx { () => ("rcx") }
+        #[allow(unused)] macro_rules! caller_vmctx { () => ("rdx") }
```

Comment:
It's probably ok to jettison this now

Reply:
Eh, I'd rather just keep it around until we completely remove the asm trampolines.